app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+// import PropTypes from "prop-types"
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.